evalai/challenges.py

```diff
@@ -264,18 +283,136 @@ def submit(ctx, file, annotation, large, public, private):
+                        if attribute_required:
+                            attribute_name = attribute_name + '*'
+                        value = None
+                        message = "{} ({})".format(
+                            attribute_name, attribute_description
+                        )
+                        if attribute_type == "text":
+                            while True:
+                                value = click.prompt(
+                                    style(message, fg="yellow"),
+                                    type=str,
+                                    default="",
+                                )
+                                if not attribute_required or value != "":
+                                    break
+                                echo(
+                                    "Error: {} is a required field".format(
+                                        attribute_name)
+                                )
+                        if attribute_type == "boolean":
+                            while True:
+                                value = click.prompt(
+                                    style(message, fg="yellow"), type=bool, default=""
+                                )
+                                if not attribute_required or value != "":
+                                    break
+                                echo(
+                                    "Error: {} is a required field".format(
+                                        attribute_name)
+                                )
+                        if attribute_type == "radio":
+                            while True:
+                                value = click.prompt(
+                                    style(
+                                        "{}: Choices:{}".format(
+                                            message, attribute["options"]
+                                        ),
+                                        fg="yellow",
+                                    ),
+                                    type=click.Choice(attribute["options"]),
+                                    default=""
+                                )
+                                if not attribute_required or value != "":
+                                    break
+                                echo(
+                                    "Error: {} is a required field".format(
+                                        attribute_name)
+                                )
+                        if attribute_type == "checkbox":
+                            optionChosen = True
+                            while optionChosen:
+                                value = []
+                                choices = click.prompt(
+                                    style(
+                                        "{}: Choices(0 or more separated by comma):{}".format(
```

Comment:
same here. And change `Choices(0 or more separated by comma)` to `Choices(separated by comma)`